src/views/AuthenticationWrapper/DevSecurity.tsx

```diff
@@ -106,9 +118,10 @@ const DevSecurity = ({ children }: ParentComponentProps) => {
             minLength={4}
             required
             value={euaId}
-            onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
-              setEuaId(e.target.value.toUpperCase())
-            }
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+              e.persist();
```

Comment:
I needed this `e.persist()` at some point, but we might not need it anymore?

Reply:
This prevents the event from being reused later on, but since we just grab the target's value here it's probably no longer needed.